src/components/LocalIdentityModal/LocalIdentityModal.js

```diff
@@ -4,34 +4,28 @@ import styled from 'styled-components'
 import {
   Button,
   EscapeOutside,
+  Modal,
   TextInput,
   breakpoint,
   textStyle,
   useTheme,
 } from '@aragon/ui'
-import { ModalContext } from '../ModalManager/ModalManager'
```

Comment:
Are we able to remove the old ModalManagers and etc. now?

Reply:
I think so, we should double check if no other component is using them.